administrator/language/en-GB/com_workflow.ini

```
@@ -104,7 +104,8 @@ COM_WORKFLOW_WORKFLOWS_TABLE_CAPTION="Workflows"
 COM_WORKFLOW_WORKFLOW_NOTE="Note"
 JLIB_HTML_PUBLISH_ITEM="Enable"
 JLIB_HTML_UNPUBLISH_ITEM="Disable"
-; Deprecated 5.0. The following strings have never been used
+
+; All the following strings are deprecated and will be removed with 5.0
```

deprecated since 4.2 or 4.3 ?

```suggestion
; The following strings are deprecated and will be removed in 5.0
```

[EDIT I see that `with` is correct, see #40493; mostly is used `in`]

---

Just copied from #40469 (to be consistent). 😉 
All other language string deprecations have only the version when they will be removed.